### PR TITLE
fix(builtin): process/browser should resolve from browserify

### PR DESCRIPTION
**Pull request by alexeagle**
Previously I marked the action local:1 but this masked a real failure in downstream project

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: internal/npm_install/browserify-wrapped.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 const child_process = require('child_process');
 
-const DEBUG = false;
+const DEBUG = true;
```
Comment (gregmagolan) [inline internal/npm_install/browserify-wrapped.js]: switch back to false

File: .bazelrc [hunk 2]
```diff
@@ -11,3 +11,16 @@ build --host_force_python=PY2
 # Opt-in to upcoming breaking changes
 build --incompatible_use_python_toolchains
 common --incompatible_string_join_requires_strings
+
+###############################
+# Remote Build Execution support
+# Turn on these settings with
+#  --config=remote
```
Comment (gregmagolan) [inline .bazelrc]: should we turn on remote for CI?

Comment (alexeagle) [reply .bazelrc]: it's already enabled on the buildkite build, that's why I found the issue to begin with. (but local:1 was the wrong solution)

File: internal/npm_install/browserify-wrapped.js (hunk 1 above)
Comment (alexeagle) [reply internal/npm_install/browserify-wrapped.js]: oops thx

**Pull request merged**